mne/viz/tests/test_topomap.py

```diff
@@ -138,13 +138,7 @@ def test_plot_topomap_animation():
     plt.close('all')
 
     # Test plotting of fnirs types
-    montage = make_standard_montage('biosemi16')
-    ch_names = montage.ch_names
-    ch_types = ['eeg'] * 16
-    info = create_info(ch_names=ch_names, sfreq=20, ch_types=ch_types)
-    evoked_data = np.random.randn(16, 30)
-    evokeds = EvokedArray(evoked_data, info=info, tmin=-0.2, nave=4)
-    evokeds.set_montage(montage)
+    evokeds = _gen_nirs_data()
```

Comment:
it's weird that _gen_nirs_data gets you some data with no NIRS channel. Can you put the set_channel_types in the private function _gen_nirs_data?

Reply:
Yes that was an odd choice. Should be fixed now.

Reply:
... then in this func you just do:
```
def test_plot_topomap_animation(fnirs_evoked):
    ...
```
And `fnirs_evoked` will be the one returned by the fixture